captum/optim/_param/image/images.py

```diff
@@ -254,10 +307,9 @@ def forward(self):
         image = self.parameterization()
         image = self.decorrelate(image)
         image = image.rename(None)  # TODO: the world is not yet ready
-        return torch.sigmoid_(image)
+        return CudaImageTensor(torch.sigmoid_(image))
```

Comment:
Does this have to be `CudaImageTensor` ? Can't it be regular `ImageTensor` that can be on any device ?

Reply:
I think we can have a regular ImageTensor, but I had some issues when switching between cuda and cpu. Will need some time to figure it out, will merge this for the time being.